dbms/src/Operators/ExchangeReceiverSourceOp.cpp

```diff
@@ -0,0 +1,108 @@
+// Copyright 2023 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <Operators/ExchangeReceiverSourceOp.h>
+
+namespace DB
+{
+void ExchangeReceiverSourceOp::operateSuffix()
+{
+    LOG_DEBUG(log, "finish read {} rows from remote", total_rows);
```

Comment:
change to LOG_INFO?

Reply:
ok, the log in `TiRemoteBlockInputStream` changed also.